src/utilities/utilities.js

```diff
@@ -568,3 +568,38 @@ export function valid_range(range, [min, max]) {
   if (range === undefined) return [];
   return range.map((n) => (n >= min && n <= max ? n : NaN));
 }
+
+// Met values of typical tasks.
```

Comment:
Can you include JSDoc comments, please? [see here](https://pythermalcomfort.readthedocs.io/en/latest/reference/pythermalcomfort.html#reference-values-clo-and-met)
Remember to add `@public` `@docname` and `@memberof` so it shows correctly in the docs